src/main/java/com/exadel/coolDesking/common/exception/ExceptionAdvice.java

```diff
@@ -0,0 +1,77 @@
+package com.exadel.coolDesking.common.exception;
+
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Slf4j
+@RestControllerAdvice
+public class ExceptionAdvice {
+
+    @ExceptionHandler(NotFoundException.class)
+    @ResponseStatus(HttpStatus.NOT_FOUND)
+    public ResponseEntity<ExceptionResponse> handleNotFoundException(NotFoundException exception){
+        Logger logger = LoggerFactory.getLogger(exception.getType());
```

Comment:
@Slf4j annotation allows you not to create logger manually. Logger will be available through **log** field;
```suggestion
        log.info("Your message");
```